lib/class-wp-duotone-gutenberg.php

```diff
@@ -32,10 +32,6 @@
  * @since 6.3.0
  */
 
-if ( class_exists( 'WP_Duotone_Gutenberg' ) ) {
```

Comment:
These have Gutenberg prefixes so there's not need to guard. They are not meant to be included with this name in core.